woocommerce/woocommerce-ios
# Update momentary selection state row background colours

Closes: #11660


## Description

- Added `configureDefaultBackgroundConfiguration()`, `updateDefaultBackgroundConfiguration()` and `listSelectedBackground` for default cell selected background color

## Testing instructions


Open these parts of the app and check the cell selected background color, it should be `Woo Purple shade0`:
- Coupons
- Reviews
- Payments Menu
- Settings Menu
- Product forms
- Product forms bottom sheets
- Store switcher

## Screenshots


Products list example:
![Simulator Screenshot - iPhone 15 Pro Max - 2024-01-23 at 17 39 27](https://github.com/woocommerce/woocommerce-ios/assets/6242034/2ac07b40-712b-449d-9d75-9d0200a5516d)

---
